File: solucoes/2013/nivel jr/fase 1/py/saldo.py
N, S = map(int, input().split())
i = 0
saldo_dias = [S]
gasto = S
while i < N:
    i += 1
    A = int(input())
    S += A
    saldo_dias.append(S)

saldo_dias.sort()
print(saldo_dias[0])

File: solucoes/2013/nivel jr/fase 1/js/saldo.js
var N, S, A 
scanf("%d%d", "N", "S")
var saldo_dias = [S]

function ordenar(a, b){
	return a-b
}

for(var i = 0; i < N; i++){
    scanf("%d", "A")
    S += A
    saldo_dias.push(S)
}

saldo_dias.sort(ordenar)

printf("%d", saldo_dias[0])
